### ISSUE TITLE
Cannot use Protobuf and ALL_ALL_PARAMETERS_NEW

### ISSUE THREAD

**Issue by lbarman** [dedis/prifi#98]
Failing tests in `prifi-lib/net/`.
We have two problems:
1) I don't know how to call protobuf.Encode correctly (fails on a simple message, see `TestEncodeDecodeStdMessage`)
2) Protobuf does not decode correctly a `map[string]Interface`

**Comment by lbarman** [comment]
More details:
Relay sends
```
&{ForceParams:true Params:map[UpCellSize:{Data:0} NextFreeTrusteeID:{Data:0} NextFreeClientID:{Data:0} NClients:{Data:1} NTrustees:{Data:1} StartNow:{Data:true}]}
```
Trustee receives
```
{ForceParams:true Params:map[NClients:{Data:<nil>} NTrustees:{Data:<nil>} StartNow:{Data:<nil>} UpCellSize:{Data:<nil>} NextFreeTrusteeID:{Data:<nil>}]}
```

**Comment by lbarman** [comment]
Fixed cacc406bc388f55106d9c72075694ce5108adb1d
